app/models/manageiq/providers/network_manager.rb

```diff
@@ -86,5 +86,14 @@ def self.supported_types_and_descriptions_hash
     def name
       "#{parent_manager.try(:name)} Network Manager"
     end
+
+    def self.cloud_network_managers
+      # filter out openstack undercloud
+      ids = ::ManageIQ::Providers::Openstack::NetworkManager.joins(:parent_manager).where(:parent_managers_ext_management_systems=>{:type=>'ManageIQ::Providers::Openstack::CloudManager'}).pluck(:id)
```

Comment:
this will make 3 queries, to probably have rbac make yet another.
Maybe this does not matter as the number of managers is probably small.

Would it be possible to build the `type` clause to include managers from openstack and redhat?